compiler/src/codegen/transl_anf.re

```diff
@@ -589,7 +596,14 @@ let rec compile_comp = (env, c) => {
       )
     | CLambda(args, body) =>
       MAllocate(MClosure(compile_lambda(env, args, body, c.comp_loc)))
-    | CApp(f, args) =>
+    | CApp(f, args, true) =>
+      /* TODO: Utilize MCallKnown */
```

Comment:
What does this mean?

Reply:
It should actually say use `MReturnCallKnown` 😛

But it means that we should use `return_call` when we know what the function we're calling is instead of `return_call_indirect`. We just don't have that information available in the compiler right now. There's a similar comment for non-tail `CApp` (which is where I copied this from).